### ISSUE TITLE
Website mode seems to prohibit persistent Julia kernels

### ISSUE THREAD

**Issue by jkrumbiegel** [quarto-dev/quarto-cli#1613]
### Bug description

I generated a blog project with the default quarto command on MacOS 12.3. Then I added some Julia code to `posts/welcome/index.qmd` such that it reads:

````qmd
---
title: "Welcome To My Blog"
author: "Tristan O'Malley"
date: "2022-07-27"
categories: [news]
jupyter: julia-1.7
---

This is the first post in a Quarto blog. Welcome!

![](thumbnail.jpg)

Since this post doesn't specify an explicit `image`, the first image in the post will be used in the listing page of posts.

### Dependencies:

```{julia}
using Pkg
1+2
Pkg.status()
```
````

When I run `quarto preview . --execute-debug` from the root folder and make a change and save, I see this each time:

```
[1/2] posts/welcome/index.qmd
- Checking for existing kernel
- No existing kernel found

Starting julia-1.7 kernel...- Executing notebook with oneshot kernel
```

The start of the kernel takes 30+ seconds each time so that's not great of course.
If I do `quarto preview posts/welcome/index.qmd --execute-debug` the same behavior occurs.

However, when I remove the `_quarto.yml` in the root folder and then do `quarto preview posts/welcome/index.qmd --execute-debug`, the output is:

```
- Connecting to kernel

Starting julia-1.7 kernel...2022-07-31 20:31:26,974 - root - Level 25 - starting notebook server subprocess
2022-07-31 20:31:26,974 - root - Level 25 - starting notebook server subprocess
2022-07-31 20:31:27,161 - root - Level 25 - running notebook server subprocess
2022-07-31 20:31:27,162 - root - Level 25 - creating notebook server (tcp: /Users/.../Library/Caches/quarto/jt/4c2f75fc67df95bfca68)
2022-07-31 20:31:27,166 - root - Level 25 - notebook server bound to port 61231
2022-07-31 20:31:27,324 - root - Level 25 - handling server request
- Kernel connection successful
- Sending execute command to kernel
- Execute command sent, reading response
2022-07-31 20:31:27,326 - root - Level 25 - executing notebook
Done

Executing 'index.ipynb'
  Cell 1/1...Done

- Server request complete
```

After that, I can make changes and the update is very fast, the existing kernel is found.
I assumed it would be no problem to use blog mode with persistent Julia kernels, am I doing something wrong?

### `quarto check` Output

```bash
[✓] Checking Quarto installation......OK
      Version: 1.0.37
      Path: /Applications/quarto/bin

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.8.9
      Path: /Library/Developer/CommandLineTools/usr/bin/python3
      Jupyter: 4.11.1
      Kernels: julia-1.7, python3

[✓] Checking Jupyter engine render....OK

[✓] Checking R installation...........(None)

      Unable to locate an installed version of R.
      Install R from https://cloud.r-project.org/
```

### `quarto tools check` Output

```
[✓] Inspecting tools

Tool         Status            Installed     Latest  
chromium     Not installed     ---           869685  
tinytex      Not installed     ---           v2022.07
```

### Checklist

- [X] [formatted your issue](https://yihui.org/issue/#please-format-your-issue-correctly) so it is easier for us to read?
- [X] included a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] documented the quarto version you're running, by pasting the output from running `quarto check` in the "Quarto Check Output" text area?
- [X] documented the version of the quarto tools you're running, by providing the output from running `quarto tools check` in the "Quarto Tools Check Output" text area?
- [X] documented the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] documented which operating system you're running? If on Linux, please provide the specific distribution as well.
- [X] upgraded to the latest version, including your versions of R, the RStudio IDE, and relevant R packages?

**Comment by jjallaire** [comment]
Hi @jkrumbiegel thanks for reporting this and sorry about the delay in responding! This bug should be resolved here (https://github.com/quarto-dev/quarto-cli/commit/d8da7fed1077391c79c53eee9b8309c5f2a917ff) and we backported the fix to v1.0.38 (noted you are running v1.0.37). This is also fixed in our v1.1 pre-release: https://quarto.org/docs/download/prerelease.html

**Comment by jkrumbiegel** [comment]
Thank you, I will try this out.
